tasks/centos.yml

```diff
@@ -11,6 +11,7 @@
     - php{{ php_version | replace('.','') }}-php-pgsql
     - php{{ php_version | replace('.','') }}-php-pdo
     - php{{ php_version | replace('.','') }}-php-mbstring
+    - php{{ php_version | replace('.','') }}-php-xml
```

Comment:
`composer` uses json for everything so it might be coming from a project, in that case I'd recommend using #3 for XML